resources/js/router/index.ts

```diff
@@ -1,14 +1,10 @@
 import { createRouter, createWebHistory, RouteRecordRaw } from 'vue-router'
-  
+
 import authRoutes from './routes/auth'
 import dashboardRoutes from './routes/dashboard'
 import facilityRoutes from './routes/facility'
 
-const routes: RouteRecordRaw[] = [
-    ...authRoutes,
-    ...dashboardRoutes,
-    ...facilityRoutes
-]
+const routes: RouteRecordRaw[] = [...authRoutes, ...dashboardRoutes, ...facilityRoutes]
```

Comment:
Please change this formatting back as it was (disable auto formatting in VSCode if needed)